src/pages/availability-panel.tsx

```diff
@@ -58,7 +57,7 @@ const AvailabilityPanel: FC = () => {
     );
   } else if (isLoading) {
     isErrorOrIsLoading = (
-      <span className={classNames.statusMessage}>Loading...</span>
```

Comment:
We should be able to see loading message whenever we visit **/availability-panel** for the first time and whenever a task is dropped on a member or vice versa we should not be showing a loading message